src/Nerdbank.Streams.Tests/Nerdbank.Streams.Tests.csproj

```diff
@@ -31,8 +38,10 @@
     </PackageReference>
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.9.0" />
     <PackageReference Include="Moq" Version="4.10.1" />
-    <PackageReference Include="OpenCover" Version="4.6.519" />
     <PackageReference Include="System.IO.Pipes" Version="4.3.0" />
+    <PackageReference Include="coverlet.msbuild" Version="2.*">
```

Comment:
I really dislike `*` based package references because from one build to the next the dependency can change without a trace and failures or differences in results are *really* hard to track down. Can we please pin the version instead?